win32/devdoc/gballoc_ll_jemalloc.md

```diff
@@ -165,4 +167,28 @@ static void jemalloc_print_stats_callback(void* context, const char* text)
 
 **SRS_GBALLOC_LL_JEMALLOC_01_009: [** If `text` is NULL, `jemalloc_print_stats_callback` shall return. **]**
 
-**SRS_GBALLOC_LL_JEMALLOC_01_010: [** Otherwise, `jemalloc_print_stats_callback` shall print (log) `text`, breaking it does in chunks of `LOG_MAX_MESSAGE_LENGTH / 2`. **]**
\ No newline at end of file
+**SRS_GBALLOC_LL_JEMALLOC_01_010: [** Otherwise, `jemalloc_print_stats_callback` shall print (log) `text`, breaking it does in chunks of `LOG_MAX_MESSAGE_LENGTH / 2`. **]**
+
+### gballoc_ll_set_decay
+
+```c
+MOCKABLE_FUNCTION(, int, gballoc_ll_set_decay, int64_t, decay_milliseconds);
+```
+
+`gballoc_ll_set_decay` sets the dirty and muzzy pages decay time to `decay_milliseconds` milliseconds.
+
+**SRS_GBALLOC_LL_JEMALLOC_28_001: [** `gballoc_ll_set_decay` shall advance jemalloc's internal epoch counter. **]**
```

Comment:
> shall advance jemalloc's internal epoch counter [](http://example.com/codeflow?start=60&length=48)

how does it do that?

My expectation here requirement-wise would be that an API is mentioned by name with enough information so that unit tests are craftable just by reading the spec.

Reply:
Basically this goes for all requirements written here as I'd have no idea what jemalloc API they call.

Reply:
Done